### DISCUSSION TITLE
CAN I HOST HOST ON AZURE OR AWS VPS

### DISCUSSION THREAD

**Discussion by Itseric100** [postalserver/postal#2248] [Help with using Postal]
I actually found a way to host postal on azure and also amazon awz vps but the only issue i am encountering is when i send my messages, it PENDS for a while and ended up showing EXECUTION EXPIRED error. So my question is if anyone has tried to host on any of these two vps and sent some messages successfully OR if i just have to enable some sort of inbound rule on the vps even though i feel i have done most of that. THANKS IN ADVANCE.

**Comment by willpower232** [comment]
It is likely that you have to contact whoever you're hosting with to find out if you need to ask them to unblock you sending emails from your account. We use AWS but did have to fill out a reverse DNS application.

**Reply by Itseric100** [reply]
thanks so much willpower, ill reach out to them now and post the feedback here as well and ill apply for the reverse DNS application too
